plugins/org.jboss.tools.jst.web.ui/src/org/jboss/tools/jst/web/ui/WebUiPlugin.java

```diff
@@ -264,6 +264,14 @@ public static JSPTextEditor getActiveEditor() {
 		return editor;
 	}
 
+	public static JSPMultiPageEditor getActiveMultiPageEditor() {

```

Comment:
actually - I Can't seem to find where this method is actually used in this new code?